pwsh/get-vcpkg-deps.ps1

```diff
@@ -21,9 +21,11 @@ if ($IsWindows) {
     choco install nasm
 } elseif ($IsMacOS) {
     brew install nasm
-    # Uninstall this because it can result in a non-working heif plugin,
-    # but it might not be present, so silence stderr
-    brew uninstall --ignore-dependencies webp 2>$null
+    # Uninstall these, otherwise the heif plugin could reference them and
+    # end up not working, but silence stderr in case they aren't present
+    foreach ($pkgName in @('webp', 'aom', 'libvmaf')) {
```

Comment:
I can't remember if this was related to KDE Frameworks 6.8+ or a vcpkg update but somewhere along the way I found I had to remove some additional things to keep the heif plugin working.